tests/fuzz/inject_fuzzer.go

```diff
@@ -20,6 +20,8 @@ import (
 
 	fuzz "github.com/AdaLogics/go-fuzz-headers"
 
+	_ "k8s.io/apimachinery/pkg/api/apitesting"
```

Comment:
what is this for?

Reply:
shouldn't have been there. Sorry for that.